### PR TITLE
Add quick funcs

### PR DESCRIPTION
**Pull request by cpjk**
Addresses #76 
Add a dropdown for using a "quick func" which is a func that does a specific thing.

Users can use this instead of copy-pasting a function into the "Function" text box.

**_This is scrappy. Just look at the file name and it tells you what the func does. UX improvements can be done later_**

screenshot-2017-07-24-17-40-16

### CODE REVIEW

File: routes/chorals.js [hunk 1]
```diff
@@ -249,5 +275,39 @@ router.post('/edit/:choralId', function(req, res, next) {
...
+    fs.readdir(DEFAULT_FUNCS_DIR, (err, fileNames) => {
+      if (err) {
+        reject(err);
+      }
+      else {
+        let fileContentPromises = fileNames
+          .map((fileName) => readFile(DEFAULT_FUNCS_DIR + '/', fileName));
+
+        // when all files are successfully read, return their contents
+        Promise.all(fileContentPromises)
```
Comment (andreirtaylor) [inline routes/chorals.js:362]: We should move the functions from the weather folder into the top level folder if we are going to do this. We cant see them as is.

Comment (andreirtaylor) [reply routes/chorals.js:362]: Or we could look in all the folders as well. That could be interesting.

Comment (cpjk) [reply routes/chorals.js:362]: yeah, i just wanted to do this quick. didnt want to implement recursive directory traversal. just dfs but fuck it

Comment (cpjk) [reply routes/chorals.js:362]: @andreirtaylor added the weather quick funcs

**Comment by jboretsky** [comment]
Looks great :taco:

**Pull request merged**